client/src/pages/ManageSteps/DragDrop.js

```diff
@@ -0,0 +1,220 @@
+                          style={{
+                            background: snapshot.isDraggingOver
+                              ? 'lightblue'
+                              : 'transparent',
+                            padding: 4,
+                            width: '100%',
+                          }}
+                        >
+                          {column.items.map((item, index) => {
+                            return (
+                              <Draggable
+                                key={item.title + item.id}
+                                draggableId={item.title + item.id}
+                                index={index}
+                              >
+                                {(provided, snapshot) => {
+                                  return (
+                                    <div
+                                      ref={provided.innerRef}
+                                      {...provided.draggableProps}
+                                      {...provided.dragHandleProps}
+                                      onClick={() =>
+                                        history.push(
+                                          navRoutes.ADMIN.EDIT_STEP.replace(
+                                            ':id',
+                                            item.id
+                                          )
+                                        )
+                                      }
+                                      style={{
+                                        userSelect: 'none',
+                                        padding: 16,
+                                        margin: '0 0 8px 0',
+                                        minHeight: '50px',
+                                        backgroundColor: snapshot.isDragging
+                                          ? '#263B4A'
+                                          : '#456C86',
+                                        color: 'white',
+                                        ...buttonStyle,
+                                        ...provided.draggableProps.style,
+                                      }}
+                                    >
+                                      <T.P weight="bold">{item.title}</T.P>
+                                      <Icon
+                                        color="borderPrimary"
+                                        icon="forwardArrow"
+                                      />
+                                    </div>
+                                  );
+                                }}
+                              </Draggable>
+                            );
+                          })}
+                          {provided.placeholder}
+                        </div>
+                      );
+                    }}
+                  </Droppable>
+                </G.Col>
+              </G.Row>
+            </div>
+          );
+        })}
+      </DragDropContext>
+
```

Comment:
Would need to prepare for an error message to be shown here in case of saving issues